### ISSUE TITLE
TODO: support Android platform

### ISSUE THREAD

**Issue by neojou** [neojou/rtw88-usb#21]
To add Android's relative vendor commands to support Android platform

**Comment by kimocoder** [comment]
Building this driver for Android 10, and currently all PCI chipsets works out,
but USB options is missing in both Kconfig and Makefile at the moment.

will continue the strive for it soon. keep this updated.

**Comment by neojou** [comment]
Yes. If only for wifi basic function, it can run on Android platform already. But if we want to support Android vendor commands with wifi_hal, e.g. https://github.com/neojou/android_wifi_hal, and to pass Android xTS (vts/cts), some more implementations are necessary for this.

**Comment by kimocoder** [comment]
Yes, and a proper Kconfig / Makefile setup on  RTW88_USB inn order to parse it the Android way. 

Have another issue related to monitor mode, posting that in the right issue tomorrow :+1: 

Awesome work, cheers
